src/themes/index.ts

```diff
@@ -5,8 +5,14 @@ export interface BiesseTheme {
   name: string;
   color: {
     primary: string;
+    secondary?: string;
   };
   font: {
+    size?: {
+      small?: string;
+      medium?: string;
+      large?: string;
+    };
     family: string;
   };
 }
```

Comment:
In generale non farei gli attributi del tema opzionali, altrimenti c'è sempre da gestire la fallback. Per Biesse facciamo lo stesso identico tema, per ora